src/suites/cts/fences.spec.ts

```diff
@@ -0,0 +1,137 @@
+export const description = ``;
+
+import { attemptGarbageCollection } from '../../framework/collect_garbage.js';
+import { TestGroup } from '../../framework/index.js';
+
+import { GPUTest } from './gpu_test.js';
+
+export const g = new TestGroup(GPUTest);
+
+// Test default initial value.
+g.test('initial/default', t => {
+  const fence = t.queue.createFence({});
+  t.expect(fence.getCompletedValue() === 0);
+});
+
+// Test initial value.
+g.test('initial/initialValue', t => {
+  const fence = t.queue.createFence({ initialValue: 2 });
+  t.expect(fence.getCompletedValue() === 2);
+});
+
+// Promise resolves when onCompletion value is less than signal value.
+g.test('signal/less', async t => {
```

Comment:
nit: on completion less ? I think less isn't so descriptive and wouldn't mean much in logs.

Reply:
Renamed a bunch of the tests and now they're much more consistent.